header.go

```diff
@@ -72,6 +72,9 @@ func (header *Header) EqualsTo(otherHeader *Header) bool {
 	if header.Command.IsLocal() {
 		return true
 	}
+	if header.Version == 0x02 && !bytes.Equal(header.rawTLVs, otherHeader.rawTLVs) {
```

Comment:
Why does it matter what version the header is? Perhaps add a comment?

Reply:
TLVs only exist for version 2, will do.

Reply:
Added.